core/shared/src/main/scala/scalaz/zio/internal/FiberContext.scala

```diff
@@ -569,4 +599,6 @@ private[zio] object FiberContext {
 
     def Initial[E, A] = Executing[E, A](FiberStatus.Running, Nil)
   }
+
+  type FiberRefLocals = java.util.Map[FiberRef[_], (Any, FiberId)]
```

Comment:
We never use the `FiberId`, do we?

Reply:
Ahh, I totally forgot about that. Removed.